vividus-bdd-engine/src/main/java/org/vividus/bdd/converter/ComparisonRuleParameterConverter.java

```diff
@@ -21,15 +21,14 @@
 import javax.inject.Named;
 
 import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
-import org.jbehave.core.steps.ParameterConverters.FluentEnumConverter;
 import org.vividus.bdd.steps.ComparisonRule;
 
 @Named
 public class ComparisonRuleParameterConverter extends AbstractParameterConverter<ComparisonRule>
 {
-    private final FluentEnumConverter fluentEnumConverter;
+    private final FluentTrimmedEnumConverter fluentEnumConverter;
```

Comment:
FluentTrimmedEnumConverter - converter is not trimmed itself as it may seem from the name, so maybe make sense to rename it to FluentTrimmingEnumConverter?